### DISCUSSION TITLE
Why is md:w-1/3 ignored here

### DISCUSSION THREAD

**Discussion by kylescousin** [tailwindlabs/tailwindcss#6207] [General]
https://play.tailwindcss.com/dk7RHCqH0a

The lg:w-1/4 works, but then it goes straight to 1 column when making the window smaller... it should be 33.3% on md size., but it jump straight to 1 column instead of 3 on md.

**Comment by aggreggator** [comment]
Check your viewport flex usage:
https://play.tailwindcss.com/jern29bl9L

**Comment by kylescousin** [comment]
Ok, it's fine now
